tests/unit/components/sl-radio-test.js

```diff
@@ -0,0 +1,48 @@
+import{ moduleForComponent, test } from 'ember-qunit';
+
+moduleForComponent( 'sl-radio', 'Unit | Component | sl radio', {
+    unit: true
+});
+
+test( 'Correct default property values', function( assert ) {
+    const component = this.subject();
+
+    assert.strictEqual(
+        component.get( 'name' ),
+        null,
+        'Default property "name" is null'
+    );
+
+    assert.strictEqual(
+        component.get( 'label' ),
+        null,
+        'Default property "label" is null'
+    );
+
+    assert.strictEqual(
+        component.get( 'readonly' ),
+        false,
+        'Default property "readonly" is false'
+    );
+
+    assert.strictEqual(
+        component.get( 'disabled' ),
+        false,
+        'Default property "disabled" is false'
+    );
+
+    assert.strictEqual(
+        component.get( 'value' ),
+        null,
+        'Default property "value" is null'
+    );
+});
+
+test( 'Inline property sets relevant class', function( assert ) {
```

Comment:
* This test should be testing both conditions:  "radio-inline" when inline, "radio" default
* The `_dependentKeys` property should be used to confirm the correct properties are being observed.  A demo of this property's use (not in a test though approach is the same) can be viewed at http://jsbin.com/zituta/edit?html,js,output
* This computed property should be tested via calling it, such as via `this.get( 'radioType` )`.  Integration tests are about the rendering of the component and whether the template is configured as expected, unit tests are about the output of the function.